specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2019-05-01/examples/NasCluster/Get.json

```diff
@@ -0,0 +1,27 @@
+{
+  "parameters": {
+    "api-version": "2019-05-01",
+    "subscriptionId": "72526b2b-4b61-4700-8f0c-7bb7ae9b6a2d",
+    "resourceGroupName": "System.redmond",
+    "location": "redmond",
+    "nasCluster": "bc489b66-faa7-41f0-bee7-bcba258f10fb"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/72526b2b-4b61-4700-8f0c-7bb7ae9b6a2d/resourceGroups/System.redmond/providers/Microsoft.Fabric.Admin/fabricLocations/redmond/nasClusters/bc489b66-faa7-41f0-bee7-bcba258f10fb",
+        "name": "redmond/bc489b66-faa7-41f0-bee7-bcba258f10fb",
+        "type": "Microsoft.Fabric.Admin/fabricLocations/nasClusters",
+        "location": "redmond",
+        "tags": {},
+        "properties": {
+          "size": 193273528320,
+          "sizeRemaining": 20185522176,
+          "clusterName": "IsilonR430",
+          "portalUri": "https://100.85.198.239:8080/"
+        }
+      }
+    },
+    "404": {}
```

Comment:
Hello. You should change this to fix CI error

Reply:
OK